app/src/components/pages/ErrorPage.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ErrorImage from 'images/404.png';
+import Button from '@mui/material/Button';
+
+function ErrorPage() {
+  return (
+    <div className="Wraper" style={{ textAlign: 'center' }}>

```

Comment:
We should avoid using pure divs here since MUI has [containers](https://mui.com/material-ui/react-container/). Ideally you shouldn't have any pure html elements in our codebase, MUI should have their own version of everything. It's important to be consistent with our use of MUI so we have a single source of all styling, and they handle dynamic window sizes/alignment well.